src/pages/learn/digital-gardening/index.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import groq from 'groq'
+import {sanityClient} from 'utils/sanity-client'
+
+const DigitalGardening: React.FC<any> = ({data}) => {
+  return <h1>{data.title}</h1>
+}
+
+export default DigitalGardening
+
+export const digitalGardeningQuery = groq`*[_type == 'resource' && slug.current == "digital-gardening-for-developers"][0]{
```

Comment:
I also love the idea that the query lives with the page and you can import that in other places that you'd like to use it.